src/service/QuestionService.spec.js

```diff
@@ -0,0 +1,65 @@
+import { WHO_IS_THAT_POKEMON } from "./modes";
+import {
+    QuestionService
+} from "./questionService.js";
```

Comment:
`questionService.js` file is `**Q**uestionService.js`

Reply:
po prostu w imporcie plik jest z małej litery, a powinien być z wielkiej, tak samo jak nazywa się plik :D

Reply:
Zmieniłem nazwę pliku w imporcie :)